tests/unit/ops/cuda/test_cuda_backward.py

```diff
@@ -6,6 +6,7 @@
 import copy
 from torch import nn
 from deepspeed import DeepSpeedTransformerLayer, DeepSpeedTransformerConfig
+from deepspeed.accelerator import get_accelerator
```

Comment:
Can you help rename this folder from `tests/unit/ops/cuda` to `tests/unit/ops/accelerators`? Okay to do in a separate PR.

Reply:
Will rename in this PR.